.github/workflows/analysis.yml

```diff
@@ -0,0 +1,37 @@
+name: Dart CI
+
+on: [push, pull_request]
+
+jobs:
+  generator_analysis:
+    runs-on: ubuntu-latest
+    container:
+      image:  google/dart:latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        working-directory: bin/objectbox_model_generator
+        run: pub get
+      - uses: axel-op/dart_package_analyzer@stable
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          maxScoreToComment: 99.99 # only a 100 score will not post a comment.
+          relativePath: 'bin/objectbox_model_generator/'
```

Comment:
Maybe if you duplicated this block for "lib-analysis", instead of having two jobs, it would not have to pull the same image twice. But that's just a guess...

Reply:
You are right, using a single job is getting the action to build only once.